torch/quantization/fx/quantize.py

```diff
@@ -168,23 +192,74 @@ def __init__(self):
     def _qat_swap_modules(self, root):
         convert(root, mapping=get_qat_module_mappings(), inplace=True, remove_qconfig=False)
 
-    def _generate_qconfig_map(self, root, input_graph):
-        def get_qconfig(module):
-            return module.qconfig if hasattr(module, 'qconfig') else None
+    def _generate_qconfig_map(self,
```

Comment:
If we go with a dict for convert, this logic can be shared with the convert_dict to control actual quantization.

Reply:
we still convert that to dict internally. only 'module_name_regex' is left at list of tuples since order is required for that.